docs/recipes/vue.md

```diff
@@ -36,9 +38,27 @@ const Vue = require('vue');
 // Setup Vue.js to remove production tip
 Vue.config.productionTip = false;
 
+// Setup Vue files to be processed by `require-extension-hooks-vue`
+hooks('vue').plugin('vue').push();
+// Setup Vue and JS files to be processed by `require-extension-hooks-babel`
+hooks(['vue', 'js']).plugin('babel').push();
+```
+
+When you are using ES6 modules, use the dist-file of Vue. Not doing that may cause a not completely accessible (and thus not testable) Vue model.
```

Comment:
ES6 => ES2015

Reply:
This is not true. Using the hooks compiles the es6 to es2015. You can use the source components.